testing/e2e/bazel.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+function write_workspace_file() {
+  cat > "$1" << EOF
+workspace(name = "$2")
+EOF
+}
+
+function create_workspace_with_default_repos() {
+  write_workspace_file "${1:-WORKSPACE}" "${2:-wix_build_tools_tests}"
```

Comment:
The assumption is that every test starts on the `WORKSPACE` root, I tend to agree with you on that since it might be problematic (and redundant) to trace working paths in case of an issue.

I'll change it to absolute path.